src/components/commons/repository/RepositoryCard.js

```diff
@@ -11,27 +12,39 @@ type RepositoryProps = {
 type StatProps = {
     iconClass: string,
     statCount: number,
+    tooltipMessage: string,
 };
 
-const Stat = ({ iconClass, statCount }: StatProps) => (
-    <div className="stat">
+const Stat = ({ iconClass, statCount, tooltipMessage }: StatProps) => (
+    <div className="stat" data-tip={tooltipMessage}>
         <i className={`fas ${iconClass}`} />
         <span className="stat-count">{statCount}</span>
     </div>
 );
 
 const RepositoryStats = ({ repository }: RepositoryProps) => (
     <div className="repository-stats">
-        <Stat iconClass="fa-star" statCount={repository.stargazersCount} />
+        <Stat
+            iconClass="fa-star"
+            statCount={repository.stargazersCount}
+            tooltipMessage="Estrelas"
```

Comment:
É melhor inglês pra padronizar com os outros
```suggestion
            tooltipMessage="Starts"
```